repo/packages/M/marathon/9/config.json

```diff
@@ -0,0 +1,378 @@
+        },
+        "save-tasks-to-launch-timeout" : {
+          "description" : "Timeout (ms) after matching an offer for saving all matched tasks that we are about to launch. When reaching the timeout, only the tasks that we could save within the timeout are also launched. All other task launches are temporarily rejected and retried later.",
+          "type" : "integer",
+          "default" : 3000
+        },
+        "scale-apps-initial-delay" : {
+          "description" : "This is the length of time, in milliseconds, before Marathon begins to periodically attempt to scale apps",
+          "type" : "integer",
+          "default" : 15000
+        },
+        "scale-apps-interval" : {
+          "description" : "This is the length of time, in milliseconds, between task scale apps.",
+          "type" : "integer",
+          "default" : 300000
+        },
+        "ssl-keystore-password" : {
+          "description" : "Password for the keystore supplied with the `ssl_keystore_path` option. Required if `ssl_keystore_path` is supplied. May also be specified with the `MESOSPHERE_KEYSTORE_PASS` environment variable.",
+          "type" : "string"
+        },
+        "ssl-keystore-path" : {
+          "description" : "Path to the SSL keystore. HTTPS (SSL) will be enabled if this option is supplied. Requires `--ssl_keystore_password`. May also be specified with the `MESOSPHERE_KEYSTORE_PATH` environment variable.",
+          "type" : "string"
+        },
+        "store-cache" : {
+          "description" : "Enabled: Enable an in memory cache for the storage layer. Disabled: Disable the in memory cache for the storage layer ",
+          "type" : "boolean",
+          "default" : true
+        },
+        "task-launch-confirm-timeout" : {
+          "description" : "Time, in milliseconds, to wait for a task to enter the TASK_STAGING state before killing it.",
+          "type" : "integer",
+          "default" : 300000
+        },
+        "task-launch-timeout" : {
+          "description" : "Time, in milliseconds, to wait for a task to enter the TASK_RUNNING state before killing it.",
+          "type" : "integer",
+          "default" : 300000
+        },
+        "tracing" : {
+          "description" : "Enabled: Enable trace logging of service method calls Disabled: (Default) Disable trace logging of service method calls",
+          "type" : "boolean",
+          "default" : false
+        },
+        "webui-url" : {
+          "description" : "The http(s) url of the web ui, defaulting to the advertised hostname",
+          "type" : "string"
+        },
+        "zk" : {
+          "description" : "ZooKeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path",
+          "type" : "string"
+        },
+        "zk-compression" : {
+          "description" : "Enabled: Enable compression of zk nodes, if the size of the node is bigger than the configured threshold. Disabled: Disable compression of zk nodes",
```

Comment:
This description doesn't match the one of the other boolean flags. It doesn't say what's the default.